conf/method.go

```diff
@@ -0,0 +1,11 @@
+package conf
+
+type Method int
+
+const (
```

Comment:
would it make sense to have `None` or `Raw` defined as null value, so the aggregation mechanism of the first archive can be expressed?